app/src/main/res/values/strings.xml

```diff
@@ -1,5 +1,6 @@
 <resources>
     <string name="device_admin_label">onebillion</string>
     <string name="device_admin_description">Admin Device Policies Description</string>
```

Comment:
Is this required after removing device admin

Reply:
I have just set the key to false to remove device admin permission and hence I have not removed the code.